test/util.py

```diff
@@ -172,9 +105,53 @@ def make_fake_annotation(
     )
 
 
+def make_fake_diff_exp_matrix(
```

Comment:
put the functions in vaguely alphabetical order for ease of finding the one you want